client/views/chan.tpl

```diff
@@ -1,7 +1,9 @@
 {{#each channels}}
 <div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
 	<span class="badge{{#if highlight}} highlight{{/if}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
-	<button class="close" aria-label="Close"></button>
+	<span class="close-tooltip tooltipped tooltipped-w" aria-label="Close">
```

Comment:
Could you change so that it's "Leave" on channels and "Close" on queries, like the context menu says? Don't worry about lobbies, we're getting rid of the close button there in #1833.